### DISCUSSION TITLE
Welcome to RedTrainer Discussions!

### DISCUSSION THREAD

**Discussion by yzsvdu** [yzsvdu/RedTrainer#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Apokah** [comment]
Can you please update ? It's not working 2 weeks now :/
TIA

**Comment by Bircherino** [comment]
Hi, is this working as of now? Having some trouble installing it so I don't want to waste too much time if it has broken :)

**Comment by Fiereu** [comment]
Hey hey, So it seems to me that you have an issue with using mappings or bringing class, field and methods names to new game revisions. Is there anyway to message u privately?

**Reply by MatheusMigliani** [reply]
Hi, im making a personal project to update this, can you reach me anywhere to talk about this?
